semantic-convention-utils/src/test/java/org/hypertrace/semantic/convention/utils/span/SpanSemanticConventionUtilsTest.java

```diff
@@ -74,4 +80,35 @@ public void testGetURIForOtelFormat() {
     v = SpanSemanticConventionUtils.getURIForOtelFormat(e);
     assertFalse(v.isPresent());
   }
+
+  @Test
+  public void testSpanKind() {
+    Map<String, AttributeValue> attributeValueMap = Map.of(
+        RawSpanConstants.getValue(OCAttribute.OC_ATTRIBUTE_SPAN_KIND),
+        SemanticConventionTestUtil.buildAttributeValue(RawSpanConstants.getValue(OCSpanKind.OC_SPAN_KIND_SERVER)));
+    assertTrue(SpanSemanticConventionUtils.isServerSpanForOCFormat(attributeValueMap));
```

Comment:
Should we add false assertion too that its not client span,

```
assertFalse(SpanSemanticConventionUtils.isClientSpanForOtelFormat(attributeValueMap)
```

Reply:
Done